config.go

```diff
@@ -82,6 +83,23 @@ func (c *Config) InitDefault(log *zap.Logger) {
 	case httpClient:
 	default:
 		c.Client = httpClient
+		// As per https://opentelemetry.io/docs/specs/otel/protocol/exporter/#specify-protocol
+		exporterEnv := "OTEL_EXPORTER_OTLP_TRACES_PROTOCOL"
+		exporterVal := os.Getenv(exporterEnv)
+		if exporterVal == "" {
+			exporterEnv = "OTEL_EXPORTER_OTLP_PROTOCOL"
```

Comment:
Would be nice to rename these vars (consts), tracesProtocol + otlpProtocol

Reply:
they're using the same variable so that the logs can inform the reader which environment variable was used.